splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/SingleInputComponent.jsx

```diff
@@ -22,15 +22,17 @@ function SingleInputComponent(props) {
         allowList,
         placeholder = _('Select a value'),
         dependencies,
+        // eslint-disable-next-line no-unused-vars
         createSearchChoice,
```

Comment:
We dont need to ingore these failures as this is a pending change in `SingleInputComponent` for which we are blocked on splunk-ui ER ticket.